mos_tests/rabbitmq_oslo/test_restarts.py

```diff
@@ -0,0 +1,156 @@
+#    Copyright 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import logging
+from random import randint
+import re
+
+import pytest
+
+
+logger = logging.getLogger(__name__)
+
+
+def vars_config(remote):
+    """Prepare variables and different paths
+    :param remote: SSH connection point to controller.
+    """
+    config_vars = {
+        'repo': 'https://github.com/dmitrymex/oslo.messaging-check-tool.git',
+        'pkg': 'oslo.messaging-check-tool_1.0-1~u14.04+mos1_all.deb',
+        'root_path': '/root/',
+        'repo_path': '/root/oslo.messaging-check-tool/',
+        'nova_user': 'nova'}
+    config_vars['conf_file_path'] = '{}oslo_msg_check.conf'.format(
+        config_vars['repo_path'])
+    # get password of nova user (the same on all controllers)
+    cmd = "grep '^rabbit_password' /etc/nova/nova.conf | awk '{print $3}'"
+    config_vars['nova_pass'] = remote.check_call(cmd)['stdout'][0].strip()
+    return config_vars
+
+
+def install_oslomessagingchecktool(remote, **kwargs):
+    """Install tool on controller
+    https://github.com/dmitrymex/oslo.messaging-check-tool
+    """
+    cmd = ("apt-get update && "
+           "apt-get install git python-pip -y && "
+           "cd {root_path} && "
+           "git clone {repo} && "
+           "cd {repo_path} && "
+           "pip install -r requirements.txt -r test-requirements.txt && "
+           "dpkg -i {pkg} || "
+           "apt-get -f install -y".format(root_path=kwargs['root_path'],
+                                          repo=kwargs['repo'],
+                                          repo_path=kwargs['repo_path'],
+                                          pkg=kwargs['pkg']))
+    logger.debug('Install "oslo.messaging-check-tool" on controller')
+    return remote.check_call(cmd)
+
+
+def configure_oslomessagingchecktool(remote, ctrl_ips, **kwargs):
```

Comment:
I believe, that you doesn't need to use **kwargs here. Will be better to describe all function arguments.